app/components/graphs/shared.tsx

```diff
@@ -0,0 +1,113 @@
+import { formatValue } from "@/app/lib/format";
+
+export interface CustomLabelListContentProps {
```

Comment:
nit: Having both `?` and `| undefined` is redundant

Reply:
Ah sorry, I missed that previous comment. It might be the way tsconifg is set up in this project - thanks!